README.md

```diff
@@ -50,8 +50,31 @@ Initialize Intercom in your `AppDelegate.m`
 
 More instructions here: [Intercom for Android](https://github.com/intercom/intercom-android)
 
-```
-Intercom.initialize(getApplicationContext(), "your api key", "your app id");
+Your Android Application should look like:
+
+```java
+// ...
+import com.robinpowered.react.Intercom.IntercomPackage;
+import io.intercom.android.sdk.Intercom;
+
+public class MainApplication extends MultiDexApplication {
```

Comment:
What is this `MultiDexApplication`?

Reply:
Hmmm actually I don't know, it's what you get when you eject from Expo (I think multidex is used for large Android applications)